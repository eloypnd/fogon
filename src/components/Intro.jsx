import React from 'react'
import { Box, Heading, Text } from 'rebass'

const Intro = () => (
  <Box px={2} py={4}>
    <Heading mb={4}>fog√≥n</Heading>
    <Text mb={2}>
      This is a playground where I practice different tools,
      techniques and architectures involve in UI development.
    </Text>
    <Text>
      Currently is using <a href='https://reactjs.org' target='_blank' rel='noopener noreferrer'>React</a> + <a href='https://redux.js.org' target='_blank' rel='noopener noreferrer'>Redux</a>.
      Styling is done with <a href='https://jxnblk.com/rebass/props' target='_blank' rel='noopener noreferrer'>Rebass</a>.
    </Text>
  </Box>
)

export default Intro
