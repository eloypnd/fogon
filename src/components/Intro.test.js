import React from 'react'
import renderer from 'react-test-renderer'
import Intro from './Intro'

it('Intro renders correctly', () => {
  const tree = renderer
    .create(<Intro />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
