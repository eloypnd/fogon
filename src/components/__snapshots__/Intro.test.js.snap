// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro renders correctly 1`] = `
<div
  className="sc-bdVaJa bjkHRI"
>
  <h2
    className="sc-VigVT hYUSXL"
  >
    fog√≥n
  </h2>
  <div
    className="sc-gqjmRU jcYQKF"
  >
    This is a playground where I practice different tools, techniques and architectures involve in UI development.
  </div>
  <div
    className="sc-gqjmRU jcYQKF"
  >
    Currently is using 
    <a
      className="sc-htoDjs gLCdRU"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      React
    </a>
     + 
    <a
      className="sc-htoDjs gLCdRU"
      href="https://redux.js.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Redux
    </a>
    . Styling is done with 
    <a
      className="sc-htoDjs gLCdRU"
      href="https://jxnblk.com/rebass/props"
      rel="noopener noreferrer"
      target="_blank"
    >
      Rebass
    </a>
    .
  </div>
  <div
    className="sc-gqjmRU kKsieB"
  >
    You can find the source code on 
    <a
      _target="_blank"
      className="sc-htoDjs gLCdRU"
      href="https://github.com/eloypnd/fogon"
      rel="noopener noreferrer"
    >
      Github
    </a>
    .
  </div>
</div>
`;
